packages/boiler-task-selenium/src/runner/webdriverio/reporters/spec.js

```diff
@@ -0,0 +1,236 @@
+import events from 'events';
+
+
+/**
+ * Custom spec reporter adapted from https://github.com/Baltino/wdio-spec-reporter
+ *
+ */
+class SpecReporter extends events.EventEmitter {
+  constructor(baseReporter, config, options = {}) {
+    super();
+
+    this.baseReporter = baseReporter;
+    this.config = config;
+    this.options = options;
+
+    const { epilogue } = this.baseReporter;
```

Comment:
I guess what I meant is can't we just use this https://github.com/webdriverio/wdio-spec-reporter and install it as a dependency?